scipy/stats/tests/test_morestats.py

```diff
@@ -677,6 +675,21 @@ def test_moments_normal_distribution(self):
         expected = [0.011315, 1.017931, 0.05811052, 0.0754134]
         assert_allclose(moments, expected, rtol=1e-4)
 
+        # test equivalence with `stats.moment`
+        m1 = stats.moment(data, moment=1)
+        m2 = stats.moment(data, moment=2)
+        m3 = stats.moment(data, moment=3)
+        assert_allclose((m1, m2, m3), expected[:-1], atol=1e-1)
```

Comment:
So after discussion with @ev-br: this uses ``data.size=12345`` which is large enough to expect a decent match. 

``atol=1e-1`` is very little. Can you use ``atol=002`` (for first moment, which gives 0.0 for ``moment`` and 0.0113 for ``kstat``) and then tighten ``rtol`` close to as far as possible? Should be 1e-2 at least I expect.